runtimevar/awssecretsmanager/awssecretsmanager_test.go

```diff
@@ -47,12 +50,37 @@ type harness struct {
 	closer  func()
 }
 
-const maxClientRequestTokenLen = 64
+// waitForMutation uses check to wait until a mutation has taken effect.
```

Comment:
I'd document briefly here how we know from check that it needs to be retried.

Reply:
Good point, done.